spec/rails_helper.rb

```diff
@@ -3,43 +3,21 @@
 if Rails.env.production?
   abort("The Rails environment is running in production mode!")
 end
+require "simplecov-json"
+  SimpleCov.formatter = SimpleCov::Formatter::JSONFormatter
 require "coveralls"
   Coveralls.wear!
 require "codeclimate-test-reporter"
   CodeClimate::TestReporter.start
 require "spec_helper"
 require "rspec/rails"
-# Add additional requires below this line. Rails is not loaded until this point!
 require "capybara/rspec"
 require "capybara/rails"
 require "factory_girl"
 require "database_cleaner"
-# Requires supporting ruby files with custom matchers and macros, etc, in
-# spec/support/ and its subdirectories. Files matching `spec/**/*_spec.rb` are
-# run as spec files by default. This means that files in spec/support that end
-# in _spec.rb will both be required and run as specs, causing the specs to be
-# run twice. It is recommended that you do not name files matching this glob to
-# end with _spec.rb. You can configure this pattern with the --pattern
-# option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
-#
-# The following line is provided for convenience purposes. It has the downside
-# of increasing the boot-up time by auto-requiring all files in the support
-# directory. Alternatively, in the individual `*_spec.rb` files, manually
-# require only the support files necessary.
-#
-# Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
-  OmniAuth.config.mock_auth[:facebook] =
-  {
-    "provider" => "facebook",
-    "uid" => "123456",
-    "info"=>
-    { name: "Blessing Ebowe",
-      email: "blessingebowe@gmail.com",
-      profile_img: "http://graph.facebook.com/1065771400114300/picture"
-    }
-  }
-# Checks for pending migrations before tests are run.
-# If you are not using ActiveRecord, you can remove this line.
+
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.